app/src/main/java/fr/free/nrw/commons/upload/ReadFBMD.java

```diff
@@ -0,0 +1,57 @@
+package fr.free.nrw.commons.upload;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+import com.icafe4j.image.meta.*;
+import com.icafe4j.image.meta.Metadata;
+import com.icafe4j.image.meta.MetadataReader;
+import com.icafe4j.image.meta.iptc.IPTC;
+import com.icafe4j.string.StringUtils;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.Iterator;
+
+import com.icafe4j.image.meta.Metadata;
+import com.icafe4j.image.meta.MetadataType;
+import com.icafe4j.image.meta.iptc.IPTC;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.Map;
+
+import retrofit2.http.Url;
+
+public class ReadFBMD {
+    Url url;
+    ReadFBMD(Url url){
+        this.url=url;
+    }
+    public void processMetadata() throws IOException {
```

Comment:
I understand that this is currently just for testing. But in the final PR, make this method return something that can be used.

Reply:
I  am sorry for that. I'll modify accordingly :+1:

Reply:
I'll make it return a boolean value whether the Image has FBMD data or not. Will that be fine?

Reply:
Yes, that would be great.
- Does the processing take time? 
- Is it memory intensive?

Reply:
I'll verify all these properties closely and then revert back to you :)